src/Compilers/CSharp/Portable/Symbols/FunctionPointers/FunctionPointerMethodSymbol.cs

```diff
@@ -0,0 +1,218 @@
+            => Hash.Combine(ReturnType, Hash.Combine(CallingConvention.GetHashCode(), RefKind.GetHashCode()));
+
+        internal override CallingConvention CallingConvention { get; }
+        public override bool ReturnsVoid => ReturnTypeWithAnnotations.IsVoidType();
+        public override RefKind RefKind { get; }
+        public override TypeWithAnnotations ReturnTypeWithAnnotations { get; }
+        public override ImmutableArray<ParameterSymbol> Parameters =>
+            _parameters.Cast<FunctionPointerParameterSymbol, ParameterSymbol>();
+        // PROTOTYPE(func-ptr): Implement custom modifiers
+        public override ImmutableArray<CustomModifier> RefCustomModifiers => throw new NotImplementedException();
+        public override MethodKind MethodKind => MethodKind.FunctionPointerSignature;
+
+        internal override DiagnosticInfo? GetUseSiteDiagnostic()
+        {
+            if (_lazyUseSiteDiagnostic == CSDiagnosticInfo.EmptyErrorInfo)
+            {
+                DiagnosticInfo? info = null;
+                CalculateUseSiteDiagnostic(ref info);
+                _lazyUseSiteDiagnostic = info;
+            }
+
+            return _lazyUseSiteDiagnostic;
+        }
+
+        internal bool GetUnificationUseSiteDiagnosticRecursive(ref DiagnosticInfo? result, Symbol owner, ref HashSet<TypeSymbol> checkedTypes)
+        {
+            return ReturnType.GetUnificationUseSiteDiagnosticRecursive(ref result, owner, ref checkedTypes)
+                || GetUnificationUseSiteDiagnosticRecursive(ref result, RefCustomModifiers, owner, ref checkedTypes)
+                || GetUnificationUseSiteDiagnosticRecursive(ref result, Parameters, owner, ref checkedTypes);
+        }
+
+        public override bool IsVararg => false; // PROTOTYPE(func-ptr): Varargs
+
+        public override Symbol? ContainingSymbol => null;
+        // Function pointers cannot have type parameters
+        public override int Arity => 0;
+        public override ImmutableArray<TypeParameterSymbol> TypeParameters => ImmutableArray<TypeParameterSymbol>.Empty;
+        public override bool IsExtensionMethod => false;
+        public override bool HidesBaseMethodsByName => false;
+        public override bool IsAsync => false;
+        public override ImmutableArray<MethodSymbol> ExplicitInterfaceImplementations => ImmutableArray<MethodSymbol>.Empty;
+        public override Symbol? AssociatedSymbol => null;
+        public override ImmutableArray<Location> Locations => ImmutableArray<Location>.Empty;
+        public override ImmutableArray<SyntaxReference> DeclaringSyntaxReferences => ImmutableArray<SyntaxReference>.Empty;
+        public override Accessibility DeclaredAccessibility => Accessibility.NotApplicable;
+        public override bool IsStatic => true;
+        public override bool IsVirtual => false;
+        public override bool IsOverride => false;
+        public override bool IsAbstract => false;
+        public override bool IsSealed => true;
```

Comment:
>true [](start = 41, length = 4)

false? It is not abstract, not virtual, not override. How could it be marked sealed?